app/js/services/ressourceService.js

```diff
@@ -24,6 +24,10 @@ angular.module('ddsApp').factory('RessourceService', function() {
                     periode: periode,
                     montant: montantMensuelEtale
                 });
+                if (!individu.ressourcesApproxAnnuel) {
+                    individu.ressourcesApproxAnnuel = {};
```

Comment:
A more idiosyncratic expression in JS would be `individu.ressourcesApproxAnnuel = individu.ressourcesApproxAnnuel || {};`